CorsixTH/Src/th_gfx.h

```diff
@@ -484,129 +489,129 @@ class THAnimationManager
+    void set_parent(animation *pParent);
 
     void tick();
-    void draw(THRenderTarget* pCanvas, int iDestX, int iDestY);
-    bool hitTest(int iDestX, int iDestY, int iTestX, int iTestY);
-    void drawMorph(THRenderTarget* pCanvas, int iDestX, int iDestY);
-    bool hitTestMorph(int iDestX, int iDestY, int iTestX, int iTestY);
-    void drawChild(THRenderTarget* pCanvas, int iDestX, int iDestY);
-    bool hitTestChild(int iDestX, int iDestY, int iTestX, int iTestY);
-
-    THLinkList* getPrevious() {return m_pPrev;}
-    size_t getAnimation() const {return m_iAnimation;}
-    bool getMarker(int* pX, int* pY);
-    bool getSecondaryMarker(int* pX, int* pY);
-    size_t getFrame() const {return m_iFrame;}
-    int getCropColumn() const {return m_iCropColumn;}
-
-    void setAnimation(THAnimationManager* pManager, size_t iAnimation);
-    void setMorphTarget(THAnimation *pMorphTarget, unsigned int iDurationFactor = 1);
-    void setFrame(size_t iFrame);
-
-    void setSpeed(int iX, int iY) {speed.x = iX, speed.y = iY;}
-    void setCropColumn(int iColumn) {m_iCropColumn = iColumn;}
+    void draw(render_target* pCanvas, int iDestX, int iDestY);
+    bool hit_test(int iDestX, int iDestY, int iTestX, int iTestY);
+    void draw_morph(render_target* pCanvas, int iDestX, int iDestY);
+    bool hit_test_morph(int iDestX, int iDestY, int iTestX, int iTestY);
+    void draw_child(render_target* pCanvas, int iDestX, int iDestY);
+    bool hit_test_child(int iDestX, int iDestY, int iTestX, int iTestY);
+
+    THLinkList* get_previous() {return m_pPrev;}
+    size_t get_animation() const {return animation_index;}
+    bool get_marker(int* pX, int* pY);
+    bool get_secondary_marker(int* pX, int* pY);
+    size_t get_frame() const {return frame_index;}
+    int get_crop_column() const {return crop_column;}
+
+    void set_animation(animation_manager* pManager, size_t iAnimation);
+    void set_morph_target(animation *pMorphTarget, unsigned int iDurationFactor = 1);
+    void set_frame(size_t iFrame);
+
+    void set_speed(int iX, int iY) {speed.x = iX, speed.y = iY;}
+    void set_crop_column(int iColumn) {crop_column = iColumn;}
 
     void persist(LuaPersistWriter *pWriter) const;
     void depersist(LuaPersistReader *pReader);
 
-    THAnimationManager* getAnimationManager(){ return m_pManager;}
+    animation_manager* get_animation_manager(){ return manager;}
 private:
-    THAnimationManager *m_pManager;
-    THAnimation* m_pMorphTarget;
-    size_t m_iAnimation; ///< Animation number.
-    size_t m_iFrame;     ///< Frame number.
+    animation_manager *manager;
+    animation* morph_target;
+    size_t animation_index; ///< Animation number.
+    size_t frame_index;     ///< Frame number.
     union {
         struct {
-            //! Amount to change m_iX per tick
+            //! Amount to change x per tick
             int x;
-            //! Amount to change m_iY per tick
+            //! Amount to change y per tick
             int y;
```

Comment:
This becomes ambiguous, the comment points to a different `y` than the variable `y` here, but that difference disappears with your renaming. Arguably, `y` here is wrong, I'd make it `dy` at least, or `dy_per_tick` or so

Reply:
I'll take a second look at this whole struct. Seems like it confuses what it is with how it's used.